cloud-build/cloudbuild.yaml

```diff
@@ -0,0 +1,7 @@
+steps:
+# Create docker image from regular Dockerfile
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/google.com/hadoop-cloud-dev/init-actions-image', '-f', 'cloud-build/Dockerfile', '.']
```

Comment:
Is it possible to have the same path for `args` and `name` tags?

Reply:
The name refers to the base image that is being used and that has an entrypoint. In this case, gcr.io/cloud-builders/docker contains docker functionality.

Reply:
Sorry for confusion, I meant `name` in a line below:
```
- name: 'gcr.io/google.com/cloud-dataproc-ci/init-actions-image'
```

Can we change `args` line to use `cloud-dataproc-ci` in the path:
```
  args: ['build', '--tag=gcr.io/google.com/cloud-dataproc-ci/init-actions-image', '-f', 'cloud-build/Dockerfile', '.']
```